lte/gateway/c/core/oai/tasks/nas/api/mme/mme_api.hpp

```diff
@@ -169,7 +169,11 @@ status_code_e mme_api_notify_new_guti(const mme_ue_s1ap_id_t ueid,
 status_code_e mme_api_new_guti(const imsi_t* const imsi,
                                const guti_t* const old_guti, guti_t* const guti,
                                const tai_t* const originating_tai,
-                               tai_list_t* const tai_list);*/
+                               tai_list_t* const tai_list);
```

Comment:
Remove extern from this file because most of the functions would have already defined in cpp files

Reply:
This is not under extern, it is temporarily moved to emm_headers.hpp file. This file will be removed after all the files are moved to c++

Reply:
Could you add comment with mentioning github issue id i.e https://github.com/magma/magma/issues/13096

Reply:
Comment with issue id is already added